test/libsolidity/semanticTests/inheritance/inherited_function_calldata_calldata_interface.sol

```diff
@@ -25,3 +25,5 @@ contract B {
 // compileViaYul: also
 // ----
 // g() -> 42
+// gas irOptimized: 107179
+// gas legacy: 117797
```

Comment:
Why are some contracts missing `legacyOptimized` or `irOptimized`?

Reply:
If they are less then 100K gas, as that was original threshold for including gas expectation

Reply:
Ah, this is rather confusing then. Ideally if any of the runs hit the threshold then all of them should be included in the file. But due to the implementation this may be hard to achieve.

Reply:
I think it is possible to add that upgrade with not too much of effort. If you think it is useful, I'll implement it in that way.

Reply:
For optimized it is easier to add both in case that one went over 100k, as both of them run in same setup of soltest.
maybe it is worth trying maling this better